.formatter.exs

```diff
@@ -2,7 +2,7 @@
   import_deps: [:phoenix],
   inputs: [
     "*.{ex,exs}",
-    "{config,lib,test,priv}/**/*.{ex,exs}",
+    "{config,lib,test,priv,libs}/**/*.{ex,exs}",
```

Comment:
You should not add the libs dir in the formatter since there is one in the sub projects:

```suggestion
    "{config,lib,test,priv}/**/*.{ex,exs}",
```

Reply:
@taorepoara Yes but when you are editing a lib from the server workspace in vscode, the formatter does not apply to it as it takes the formatter of the server folder.

Are you sure that you want to remove this as it will force devs to open the lib in a new workspace in order to get the formatting ?

Reply:
I removed it as it caused some errors in the CI.